### ISSUE TITLE
No colorization / formatting for better interop with other programs

### ISSUE THREAD

**Issue by ajcrites** [apex/apex#293]
I've actually already written a PR for this but saw that I should open an issue first. Whoops.

At any rate, it would be handy to have a `--no-color` flag for the apex command that prevents colorizing output so you can pipe it to other programs. `apex list` is useful to get function names and then pipe it through an awkward `sed` to strip out the colors. This flag would help.

We could also use something like isatty to format the output differently for pipes but I think that would constitute a separate issue.

**Comment by tj** [comment]
hmmm personally I think I would rather have a flag that provides a more "raw" format, whatever that would be, similar to --tfvars

**Comment by ajcrites** [comment]
A `--no-color` flag and formatting output for pipes are pretty common idioms, but format flags for individual commands would also be useful. I just suggest a more global flag to make this command easier to use for a wider variety of workflows; it doesn't preclude flags for more specific format of output.

**Comment by tj** [comment]
yep, that's definitely true, I guess I'm cool with it, still useful for CI that doesn't support escape sequences etc (if any of them don't anymore haha). We'll just have to do a bit of a refactor to support the other bits of code that output color as well

**Comment by ajcrites** [comment]
I tested it out and running with `--no-color` strips the escape sequences :( but I'll take another look and check out colorstring.
